### PR TITLE
SOV-1711: Add GTM Clickthrough tracking id

### PR DESCRIPTION
**Pull request by soulBit**
https://sovryn.atlassian.net/browse/SOV-1711

### CODE REVIEW

File: netlify.toml [hunk 1]
```diff
@@ -21,6 +21,7 @@
   [context.production.environment]
     REACT_APP_NETWORK = "mainnet"
     REACT_APP_GOOGLE_ANALYTICS = "GTM-KTSX2PL"
+    REACT_APP_GOOGLE_ANALYTICS_CLICKTHROUGH = "GTM-5MWCJN3"
```
Comment (tiltom) [inline netlify.toml:24]: Why is this different than in local env files?

Comment (soulBit) [reply netlify.toml:24]: thanks, forgot to copy over correct id there - updated now

File: .env.testnet [hunk 2]
```diff
@@ -14,6 +14,7 @@ REACT_APP_TRANSAK_API_KEY=a2d23229-58f9-4ca5-a3d4-2c3d2b1b9a81
 REACT_APP_TRANSAK_ENV=STAGING
 
 REACT_APP_GOOGLE_ANALYTICS=GTM-KTSX2PL
+REACT_APP_GOOGLE_ANALYTICS_CLICKTHROUGH=GTM-5MWCJN3
```
Comment (tiltom) [inline .env.testnet:17]: Do we want it on testnet as well (I mean both of the GTMs)?

Comment (soulBit) [reply .env.testnet]: removed both of the analytics ids from this file as discussed on Jira

**Review by tiltom — APPROVED** [review]

**Comment by soulBit** [comment]
Merged via #2498

**Pull request closed without merging**